tests/tests/test_monitor_client.py

```diff
@@ -24,13 +24,16 @@
 from email.policy import default
 from ..common_setup import monitor_commercial_setup_no_client
 
+
```

Comment:
Don't know why you need this new line :)

Reply:
I have really no idea. must be a symptom of some deeper need for blank lines. removed.